test/controllers/WhatYouOweControllerSpec.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import common.TestModels.customerInformationMax
+import org.jsoup.Jsoup
+import play.api.http.Status.OK
+import play.api.test.Helpers._
+
+class WhatYouOweControllerSpec extends ControllerBaseSpec {
```

Comment:
We will need to add a DD test once the controller has been updated but could you also review the current controller spec to see if we can cover anything else off around auth etc?